Tests/FeatherWidgets.TestIntegration/Navigation/LanguageSelectorWidgetTests.cs

```diff
@@ -141,6 +146,173 @@ public void LanguageSelectorWidget_CurrentLanguageNotIncluded()
             this.AssertLanguageLinks(pageContent, expectedLinkes, notExpectedLinkes);
         }
 
+        [Test]
+        [Multilingual]
+        [Category(TestCategories.Navigation)]
+        [Author(FeatherTeams.Team7)]
+        [Description("Verifies language selector is included in detail view of content items")]
+        public void LanguageSelectorWidget_CurrentLanguageIncludedInDetailsViewOfContentItems()
```

Comment:
The test fails because on the detail page of the news item there is another link to comments. You have to change the AssertLanguageLinks method in order to work correctly.